bundler/lib/bundler/cli/common.rb

```diff
@@ -14,6 +14,22 @@ def self.print_post_install_message(name, msg)
       Bundler.ui.info msg
     end
 
+    def self.output_fund_metadata_summary(gems_seeking_funding)
```

Comment:
Actually what I meant is that this parameter is not really needed, so I think we can remove it plus all the changes in `bundler/lib/bundler/installer.rb`, and `bundler/lib/bundler/paralell_installer.rb`.

Reply:
Whoops!